plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -107,6 +116,14 @@
       - Define external parameter for this user.
     type: bool
     default: no
+  provider:
+    description:
+      - External provider for this user.
+    type: str
```

Comment:
`provider` and `extern_uid` are also new options, so they need `version_added` as well.